resolvers/Query.js

```diff
@@ -88,7 +95,13 @@ const Query = {
     authCheck(context);
     //Ensure user exists
     const user = await User.findOne({ _id: context.userId });
-    if (!user) throw new Error('User does not exist');
+    if (!user) {
+      throw new NotFoundError(
+        requestContext.translate('user.notFound'),
```

Comment:
Can you explain what is this translate doing?

Reply:
And What about custom message translation?
Let say i want to throw error: `User access is denied`, so how we will deal with that custom translation?